mm2src/mm2_metrics/src/mm_metrics.rs

```diff
@@ -410,15 +397,14 @@ mod test {
         mm_gauge!(metrics, "rpc.connection.count", 3.0, "coin" => "KMD");
         mm_gauge!(metrics, "rpc.connection.count", 5.0, "coin" => "KMD");
 
-        let delta = 34382022725155.0 - 34381019796149.0;
+        let delta = Duration::from_secs(1);
         mm_timing!(metrics,
                    "rpc.query.spent_time",
                    // ~ 1 second
                    delta, // delta
                    "coin" => "KMD",
                    "method" => "blockchain.transaction.get");
 
-        let delta = 34384023173373.0 - 34382022774105.0;
```

Comment:
This remove delta was 2 seconds. So the test doesn't cover the case when `max != min`.
Could you please return `let delta = Duration::from_secs(2)` here?

Reply:
done